Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1,148 +1,621 @@
+        readable_output=md,
+        outputs={
+            'User': raw_data.get('users'),
+            'Metadata': {'Count': raw_data.get('page_count'),
+                         'Number': raw_data.get('page_number'),
+                         'Size': raw_data.get('page_size'),
+                         'Total': raw_data.get('total_records')}
+        },
+        raw_response=raw_data
+    )
+
+
+def zoom_create_user_command(client: Client, user_type: str, email: str, first_name: str, last_name: str) -> CommandResults:
+    user_type_num = USER_TYPE_MAPPING.get(user_type)
+    raw_data = client.zoom_create_user(user_type_num, email, first_name, last_name)
+    return CommandResults(
+        outputs_prefix='Zoom.User',
+        readable_output=f"User created successfully with ID: {raw_data.get('id')}",
+        outputs=raw_data,
+        raw_response=raw_data
+    )
+
+
+def zoom_delete_user_command(client: Client, user_id: str, action: str) -> CommandResults:
+    client.zoom_delete_user(user_id, action)
+    return CommandResults(
+        outputs_prefix='Zoom',
+        readable_output=f'User {user_id} was deleted successfully',
+    )
+
+
+def zoom_create_meeting_command(
+        client: Client,
+        user_id: str,
+        topic: str,
+        host_video: bool | str = True,
+        jbh_time: int | str | None = None,
+        start_time: str = None,
+        timezone: str = None,
+        type: str = "instant",
+        auto_record_meeting: str = "none",
+        encryption_type: str = "enhanced_encryption",
+        join_before_host: bool | str = False,
+        meeting_authentication: bool | str = False,
+        waiting_room: bool | str = False,
+        end_date_time: str | None = None,
+        end_times: int | str = 1,
+        monthly_day: int | str = 1,
+        monthly_week: int | str | None = None,
+        monthly_week_day: int | str | None = None,
+        repeat_interval: int | str | None = None,
+        recurrence_type: int | str | None = None,
+        weekly_days: int | str = 1) -> CommandResults:
+
+    # converting
+    host_video = argToBoolean(host_video)
+    join_before_host = argToBoolean(join_before_host)
+    meeting_authentication = argToBoolean(meeting_authentication)
+    waiting_room = argToBoolean(waiting_room)
+    end_times = arg_to_number(end_times)
+    monthly_day = arg_to_number(monthly_day)
+    monthly_week = arg_to_number(monthly_week)
+    monthly_week_day = arg_to_number(monthly_week_day)
+    repeat_interval = arg_to_number(repeat_interval)
+    weekly_days = arg_to_number(weekly_days)
+
+    num_type = 1  # "instant"
+    if type == "scheduled":
+        num_type = 2
+    elif type == "recurring meeting with fixed time":
+        num_type = 8
+
+    # argument checking
+    # for arguments that have a default value, i use demisto.args to trigger an exception if user entered a value.
+    args = demisto.args()
+    if type == "instant" and (timezone or start_time):
```

Comment:
Good practice is that "instant" should be const as if tomorrow zoom changes it to "instant meeting", for example, you will have to find all the occurrences in the code and change it. very hard to maintain.

Reply:
Zoom API knows only numbers, "instant" is 1. so i dont care if they change the name.
and if they change the number, they may change 2 and 3 as well, so i'm not sure what you want we to do.

Reply:
I want you to change it to const, as it is a better practice not to have such strings in your code, and easier to maintain for the next developer to come.